resources/views/backend/books/edit.blade.php

```diff
@@ -1,81 +1,124 @@
-      <li class="active">{{ __('Books') }}</li>
+      <li class=""><a href="">{{ __('books.books') }}</a></li>
+      <li class="active">{{ __('books.edit_book') }}</li>
     </ol>
   </section>
   <section class="content">
       <div class="row">
         <div class="col-md-12">
           <div class="box box-primary">
-            <form action="{{ route('books.store') }}" method="POST" enctype="multipart/form-data">
+            <form action="{{route('books.update', 2)}}" role="form" method="POST" enctype="multipart/form-data">
               {{ csrf_field() }}
+              {{ method_field('PUT') }}
               <div class="box-body">
-                <div class="form-group">
-                  <label>{{ __('List Category') }}</label>
-                  <div class="row">
-                    <div class="col-xs-2">
-                      <select class="form-control" name="category" >
-                        <option value="">option 1</option>
-                        <option value="">option 2</option>
-                        <option value="">option 3</option>
-                        <option value="">option 4</option>
-                        <option value="">option 5</option>
-                      </select>
-                    </div>
-                  </div>
-                </div>
-                <div>
-                  <div class="row">
-                    <div class="col-xs-6">
-                      <div class="form-group">
-                        <label for="InputTitle">{{ __('Title') }}</label>
-                        <input type="text" id="InputTitle" class="form-control" name="title" placeholder="{{ __('Title') }}">
-                      </div>
-                      <div class="form-group">
-                        <label for="InputPrice">{{ __('Price') }}</label>
-                        <input type="text" id="InputPrice" class="form-control" name="price" placeholder="{{ __('Price') }}">  
-                      </div>
-                      <div class="form-group">
-                        <label for="InputFromPerson">{{ __('From Person') }}</label>
-                        <input type="text" id="InputFromPerson" class="form-control" name="from_person" placeholder="{{ __('ID Donator') }}">   
-                      </div>
-                      <div class="form-group">
-                        <label for="InputDescription">{{ __('Description') }}</label>
-                        <textarea class="textarea" id="InputDescription" placeholder="Place some text here"
-                          style="width: 100%; height: 200px; font-size: 14px; line-height: 18px; border: 1px solid #dddddd; padding: 10px;"></textarea>
+                <div class="row">
+                  <div class="col-xs-12">
+                    <div class="form-group">
+                      <div class="row">
+                        <div class="col-xs-12">
+                          <label for="exampleInputTitle">{{ __('books.title') }}</label>
```

Comment:
Cái thuộc tính `for` của thẻ `label` nó liên kết với `id` của thẻ `input`, nó phải trùng nhau, fix tất cả nhé:
1. Nếu đã có thuộc tính `for` trong thẻ `label` thì thẻ `input` tương ứng phải có `id` là giá trị của thuộc tính `for` nhé.
2. Có dùng `for` trong `label` thì phải liên kết với `id` của `input` tương ứng, không có liên kết thì remove nhé.
3. Ưu tiên phương án 1 

***Từng ký tự, kể cả dấu cách, dấu tab (convention) trong code phải hữu ích, ko thiếu, ko dư thừa. :D***
Mọi người chú ý nhé. Trước anh nhắc phần này rồi thì phải.